asserts/assertstest/assertstest.go

```diff
@@ -159,29 +163,28 @@ func NewAccount(db SignerDB, username string, otherHeaders map[string]interface{
 		otherHeaders["account-id"] = strutil.MakeRandomString(32)
 	}
 	if otherHeaders["display-name"] == nil {
-		otherHeaders["display-name"] = strings.ToTitle(username)
+		otherHeaders["display-name"] = strings.ToTitle(username[:1]) + username[1:]
```

Comment:
This is just a test assertion factory. Real assertions use `display-name` for the proper name of the account holder, e.g. a user's full name.